static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,10 @@
 .from-content-api pre {
     // Unset from `.from-content-api`
     word-wrap: normal;
+    background-color: #f6f9fb;
```

Comment:
Could maybe achieve them using existing colours and `lighten()` sass colour function.